_assets/javascripts/themetoggle.js

```diff
@@ -0,0 +1,26 @@
+//Theme initialize
+if (!localStorage.getItem('theme'))
+  localStorage.setItem('theme', "light");
+themer();
+
+// Theme toggle
+function toggle() {
+  if(localStorage.getItem('theme') == "light")
+    localStorage.setItem('theme', "dark");
+  else if(localStorage.getItem('theme') == "dark")
+    localStorage.setItem('theme', "light");
+  themer();
+}
+
+// Theme set
+function themer() {
+  var tone = localStorage.getItem('theme');
+  var light = document.getElementById("light");
+
+  if(tone == "dark"){
+    dark.disabled = false;
```

Comment:
i'm not sure what this `dark` variable refers to? In any case, there's a bit of a flash whenever loading a new page in dark theme.

Reply:
It looks like we are switching theme using JS and so until that JS loads, we are seeing the default theme that was set when building the website. There needs to be logic that reads localstorage before even the rendering happens.